testapi.pm

```diff
@@ -390,8 +390,9 @@ sub upload_logs {
     my ($file) = @_;
 
     bmwqemu::log_call('upload_logs', file => $file);
-    my $cmd      = "curl --form upload=\@$file ";
     my $basename = basename($file);
+    my $upname   = ref($autotest::current_test) . '-' . $basename;
```

Comment:
Can you use `$autotest::current_test->{fullname}`? It will then use `$category-$name` prefix. I think it will better address from where the logs are and I'm also afraid of tests with the same name possibly overwriting their logs. Or am I overthinking it too much?